src/components/elements/GatewayCard.tsx

```diff
@@ -45,32 +48,52 @@ const GatewayCardComponent = (props: GatewayProps) => {
             hoverable
             onClick={handleCardClick}
             title={
-              <>
-                <Text
-                  ellipsis={{
-                    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-                    tooltip: `${gatewayDetails.name}`,
-                  }}
-                  data-testid={`gateway[${index}]-details`}
-                >
-                  {gatewayDetails.name}
-                </Text>
-                <span className={cardStyles.timestamp}>
-                  Last updated{` `}
-                  {getFormattedLastSeen(gatewayDetails.lastActivity)}
-                </span>
-                <div
-                  data-testid="gateway-location"
-                  className={cardStyles.locationWrapper}
-                >
-                  <span className={cardStyles.locationTitle}>
-                    Location{` `}
-                  </span>
-                  <span className={cardStyles.location}>
-                    {formattedLocation}
+              <div className={cardStyles.headerSection}>
+                <span>
+                  <Text
+                    ellipsis={{
+                      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+                      tooltip: `${gatewayDetails.name}`,
+                    }}
+                    data-testid={`gateway[${index}]-details`}
+                  >
+                    {gatewayDetails.name}
+                  </Text>
+                  <span className={cardStyles.timestamp}>
+                    Last updated{` `}
+                    {getFormattedLastSeen(gatewayDetails.lastActivity)}
                   </span>
-                </div>
-              </>
+                  <div
+                    data-testid="gateway-location"
+                    className={cardStyles.locationWrapper}
+                  >
+                    <span className={cardStyles.locationTitle}>
+                      Location{` `}
+                    </span>
+                    <span className={cardStyles.location}>
+                      {formattedLocation}
+                    </span>
+                  </div>
+                </span>
+                <span>
+                  {gatewayDetails.cellBars ? (
+                    <Image
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      src={calculateCellSignalStrength(gatewayDetails.cellBars)}
+                      width={24}
+                      alt="Gateway cell signal strength"
```

Comment:
It’d be great if these icons also had tooltips. We could do fancy Ant ones or just give these images a `title`.

Reply:
That's a great suggestion - thanks for reminding me about the tooltip!